### PR TITLE
Add some translations

### PR DESCRIPTION
**Pull request by zdxgs**
Add some Chinese translations

### CODE REVIEW

File: translations/zh-CN.json [hunk 1]
```diff
@@ -1054,8 +1054,8 @@
 	"You are trying to run a windows .exe binary inside WSL. For better integration please use a Linux binary instead (Download at https://minikube.sigs.k8s.io/docs/start/.). Otherwise if you still want to do this, you can do it using --force": "您正在尝试在 WSL 中运行 Windows .exe 二进制文件。为了更好的集成，请改为使用 Linux 二进制文件（在 https://minikube.sigs.k8s.io/docs/start/ 下载）。如果仍然想要执行此操作，您可以使用 --force。",
 	"You are trying to run the amd64 binary on an M1 system.\nPlease consider running the darwin/arm64 binary instead.\nDownload at {{.url}}": "你正在尝试在 M1 系统上运行 amd64 二进制文件。\n请考虑改用 darwin/arm64 二进制文件。\n下载地址：{{.url}}",
 	"You are using the QEMU driver without a dedicated network, which doesn't support `minikube service` \u0026 `minikube tunnel` commands.": "",
-	"You can create one using 'minikube start'.\n\t\t": "",
-	"You can delete them using the following command(s): ": "",
+	"You can create one using 'minikube start'.\n\t\t": "您可以创建一个使用 'minikube start'.\n\t\t",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:1057]:
```suggestion
	"You can create one using 'minikube start'.\n\t\t": "您可以使用 'minikube start' 创建一个.\n\t\t",
```

File: translations/zh-CN.json [hunk 2]
```diff
@@ -1129,17 +1129,17 @@
 	"if you want to create a profile you can by this command: minikube start -p {{.profile_name}}": "如果你想创建一个配置文件，你可以执行此命令：minikube start -p {{.profile_name}}",
 	"initialization failed, will try again: {{.error}}": "初始化失败，将再次重试：{{.error}}",
 	"invalid kubernetes version": "无效的 Kubernetes 版本",
-	"ip not found": "",
+	"ip not found": "ip 找不到",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:1132]:
```suggestion
	"ip not found": "没有找到对应的 IP",
```

File: translations/zh-CN.json [hunk 3]
```diff
@@ -1129,17 +1129,17 @@
...
 	"kubectl and minikube configuration will be stored in {{.home_folder}}": "kubectl 和 minikube 配置将存储在 {{.home_folder}} 中",
-	"kubectl not found. If you need it, try: 'minikube kubectl -- get pods -A'": "",
-	"kubectl proxy": "",
-	"libmachine failed": "",
+	"kubectl not found. If you need it, try: 'minikube kubectl -- get pods -A'": "kubectl未找到。如果你需要使用它，请尝试：'minikube kubectl -- get pods -A'",
+	"kubectl proxy": "kubectl 代理",
+	"libmachine failed": "libmachine 失败",
 	"list displays all valid default settings for PROPERTY_NAME\nAcceptable fields: \n\n": "list 显示 PROPERTY_NAME 的所有有效默认设置\n可接受的字段：\n\n",
-	"list versions of all components included with minikube. (the cluster must be running)": "",
+	"list versions of all components included with minikube. (the cluster must be running)": "列出minikube包含的所有组件的版本。（群集必须正在运行）",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:1142]:
```suggestion
	"list versions of all components included with minikube. (the cluster must be running)": "列出minikube包含的所有组件的版本。（集群必须正在运行）",
```

File: translations/zh-CN.json [hunk 4]
```diff
@@ -1129,17 +1129,17 @@
...
 	"json encoding failure": "JSON 编码失败",
 	"keep the kube-context active after cluster is stopped. Defaults to false.": "在集群停止后保持 kube-context 处于活动状态。默认值为 false。",
-	"kubeadm certificates have expired. Generating new ones...": "",
+	"kubeadm certificates have expired. Generating new ones...": "kubeadm 证书已经过期。正在生成新的...",
 	"kubeadm detected a TCP port conflict with another process: probably another local Kubernetes installation. Run lsof -p\u003cport\u003e to find the process and kill it": "kubeadm 检测一个到与其他进程的 TCP 端口冲突：或许是另外的本地安装的 Kubernetes 导致。执行 lsof -p\u003cport\u003e  查找并杀死这些进程",
 	"kubectl and minikube configuration will be stored in {{.home_folder}}": "kubectl 和 minikube 配置将存储在 {{.home_folder}} 中",
-	"kubectl not found. If you need it, try: 'minikube kubectl -- get pods -A'": "",
-	"kubectl proxy": "",
-	"libmachine failed": "",
+	"kubectl not found. If you need it, try: 'minikube kubectl -- get pods -A'": "kubectl未找到。如果你需要使用它，请尝试：'minikube kubectl -- get pods -A'",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:1138]:
```suggestion
	"kubectl not found. If you need it, try: 'minikube kubectl -- get pods -A'": "kubectl 未找到。如果你需要使用它，请尝试：'minikube kubectl -- get pods -A'",
```

**Review by liangyuanpeng — COMMENTED** [review]
LGTM

**Review by spowelljr — APPROVED** [review]
Thanks for the translations and review!

**Pull request merged**